gradle.properties

```diff
@@ -25,10 +25,10 @@ stdlibSystemVersion=0.6.3-SNAPSHOT
 stdlibTimeVersion=1.0.4-SNAPSHOT
 stdlibTaskVersion=1.1.3-SNAPSHOT
 stdlibConfigVersion=1.0.3-SNAPSHOT
-stdlibFileVersion=0.6.1-SNAPSHOT
 stdlibCryptoVersion=1.0.3-SNAPSHOT
 stdlibLogVersion=1.0.3-SNAPSHOT
 stdlibReflectVersion=0.5.3-SNAPSHOT
-stdlibMimeVersion=1.0.3-SNAPSHOT
 stdlibCacheVersion=2.0.3-SNAPSHOT
+stdlibFileVersion=0.6.1-SNAPSHOT
+stdlibMimeVersion=1.0.3-SNAPSHOT
 stdlibAuthVersion=1.0.3-SNAPSHOT
```

Comment:
Shall we bump these dependency versions?

Reply:
+1. Please check the latest versions. Most of are in `*.*.4-SNAPSHOT`, except for a few modules like `time`, and `file`.

Reply:
Done